doc/python_reference.rst

```diff
@@ -904,8 +904,17 @@ Simulation
    simulate_stc
    simulate_sparse_stc
    select_source_in_label
+   stc_cosine_score
+   stc_region_localization_error
+   stc_f1_score
+   stc_precision_score
+   stc_recall_score
+   stc_roc_auc_score
+   stc_spacial_deviation
```

Comment:
stc_spacial_deviation  -> stc_spatial_deviation_error
to make explicit that lower is better? as opposed with a function that ends with _score?